src/main/java/de/chojo/repbot/analyzer/ThankType.java

```diff
@@ -1,5 +1,25 @@
 package de.chojo.repbot.analyzer;
 
 public enum ThankType {
-    FUZZY, MENTION, ANSWER, REACTION, EMBED, NO_TARGET, NO_MATCH
+    FUZZY("thankType.fuzzy"),
+    MENTION("thankType.mention"),
+    ANSWER("thankType.answer"),
+    DIRECT("thankType.direct"),
+    REACTION("thankType.reaction"),
+    EMBED("thankType.embed"),
+    NO_TARGET,
+    NO_MATCH;
+
+    private final String localizedName;
+
+    ThankType(String localizedName) {
+        this.localizedName = localizedName;
+    }
+    ThankType() {
+        localizedName = null;
+    }
```

Comment:
We should stay consistent

You can choose one:

```suggestion
    ThankType(String localizedName) {
        this.localizedName = localizedName;
    }

    ThankType() {
        this.localizedName = null;
    }
```

```suggestion
    ThankType() {
        this(null);
    }

    ThankType(String localizedName) {
        this.localizedName = localizedName;
    }
```